youtube_dl/extractor/twitcasting.py

```diff
@@ -79,3 +90,73 @@ def _real_extract(self, url):
             'uploader_id': uploader_id,
             'formats': formats,
         }
+
+
+class TwitCastingHistoryIE(InfoExtractor):
```

Comment:
I suggest you split this pull request into two: one for the bugfix and another for the new feature. So one doesn't block the other.

Reply:
As is probably obvious at this point I'm not in a hurry.